src/app/Upload.svelte

```diff
@@ -100,6 +105,11 @@
     color: var(--blue);
     padding-bottom: 1em;
   }
+
+  .download.download--disabled {
```

Comment:
What would you name it?

Reply:
@IanSeng for next time, may you please _suggest_ a better name? If a better name doesn't come to mind, either because the intent is not clear, or, you know, naming is hard, please the authors know exactly what the issue is (e.g., _why_ does this cause confusion) and that way we have an actionable way to respond to this review comment!

Reply:
hmmm my apologies for the confusion. I was confused when I first look at this line and was wondering why `download--disabled` class is having the same CSS styling as `download`. That is why I think a better naming would be great and thought it could be a mistake in the naming or my mistake.

Reply:
Confusion is okay! A helpful comment is:

> why [does] `download--disabled` class [have] the same CSS styling as `download`?

And you could add:

> Could you make a clearer name for "download"?

Also note, that I'm using BEM here, so it's `.download` with the `--disabled` modifier/state; although in all fairness, `.download` is a redundant selector (increases the specificity), so now that I think about it, I can safely remove the `.download` part of the selector! :)

Reply:
Nice one. Thank you